storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -238,20 +241,21 @@
 (defn- rmr-as-user
   "Launches a process owned by the given user that deletes the given path
   recursively.  Throws RuntimeException if the directory is not removed."
-  [conf id user path]
+  [conf id path]
+  (let [user (Utils/getFileOwner path)]
   (worker-launcher-and-wait conf
                             user
                             ["rmr" path]
                             :log-prefix (str "rmr " id))
   (if (exists-file? path)
-    (throw (RuntimeException. (str path " was not deleted")))))
+    (throw (RuntimeException. (str path " was not deleted"))))))
```

Comment:
Need to indent since we are adding the `let`.  Removing the `user` param was a good change here.